src/main/java/models/ShoppingCart.java

```diff
@@ -0,0 +1,47 @@
+package main.java.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShoppingCart {
+
+  private List<CartItem> cartItemList;
+
+  public ShoppingCart() {
+
+    cartItemList = new ArrayList<>();
+  }
+
+  public ShoppingCart(List<CartItem> cartItemList) {
+
+    this.cartItemList = cartItemList;
+  }
+
+  public int calculateTotalCost() {
+    int summation = 0;
+    for (CartItem item : cartItemList) {
```

Comment:
Essas linhas podem ser um pouco mais espaçadas pra melhorar a legibilidade

Reply:
Esse for tem uma linha só, então não da duas quebras de linha, só uma

Reply:
Sorry, não vai se repetir, métodos com somente uma linha não precisam de duas quebra de linha, entendio.